website/static/js/contribManager.js

```diff
@@ -125,7 +159,11 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
     self.profileUrl = ko.observable(contributor.url);
 
     self.canRemove = ko.computed(function(){
-        return (self.id === pageOwner.id) && !isRegistration;
+        return (self.id === pageOwner.id) && !isRegistration && !self.isAdmin;
+    });
+
+    self.canAddAdminContrib = ko.computed(function() {
+        return self.currentUserCanEdit && self.isAdmin;
```

Comment:
Are both of these conditions necessary? I would've thought that `isAdmin` encapsulates `currentUserCanEdit` (assuming `isAdmin` is referring to the current user as well).

Reply:
isAdmin refers to the user you are intending to add, not the current user.  I will add some comments, maybe change variable names to clarify this.